### DISCUSSION TITLE
Dealing with 2FA, with multiple computers, no cell phone.

### DISCUSSION THREAD

**Discussion by yanntm** [community/community#118701] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have 2FA activated, since it was forced on me in November'23 :(

I do not have a cell phone, for plenty of reasons that are good enough for me.

I did the procedure to add an authenticator extension to my browser, so I have a code when prompted to login.

Now my issue is that I have many different physical machines, not colocated, where I need to access GH. So I need to configure the authenticator extension on all of them. However, configuring one machine breaks the config on another ! I'm using recovery codes to login, but these are OTP it seems, and it is quite cumbersome to use (+ insecure since I basically mail these files to myself to have them available on all machines).

Is it possible to configure an authenticator app, on several different devices, so they are consistent/all work ? 

If so, could this be documented and added to the 2FA page ?
https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication

thank you.

**Comment by yanntm** [comment, marked as answer]
Ok, answering myself since I found a setup that works.

So after logging in, under 2FA, edit your 2FA configuration, below the QR code you'll find :
![image](https://github.com/community/community/assets/15147867/87f7dbe8-f5cc-4d89-afce-b2cc05112d87)

Click that link, and then in your 2FA browser extension (I'm using "Authenticator" but it should work on others) you setup the provider (github) and paste the key in.

So you can now mail yourself this key, so you can setup the authenticator plugin on various machines to provide the same codes. So "the trick" was to configure only once but reuse the configuration key.

Whether this whole mess actually adds a layer of "security" is left to your appreciation, it's certainly an unnecessary hassle from my POV.

**Reply by PSMJoe** [reply]
Where do I find the "2FA browser extension"?

**Comment by DataJuggler** [comment]
It was forced on me also, and now Authy is at end of life desktop, and the Edit button doesn't work. Code I can't give away for free has to have 2FA. Dumbest idea ever.
